tests/cypress/integration/tour/navigation.js

```diff
@@ -47,6 +47,14 @@ context("Navigation", () => {
     cy.get(".introjs-showElement").should("not.exist");
   });
 
+  it("should exit the tour after right btn pressed at the end", () => {
+    cy.get(".introjs-tooltiptext").contains("step one");
```

Comment:
Beautiful! Thank you

Reply:
In fact, there wasn't such error in master before adding this feature.  Possible error was  taken into account in 87fa6043b8c08d3d19d397160896ea15bd5fb467